doc/pr2_tutorials/planning/launch/move_group_interface_tutorial.launch

```diff
@@ -1,7 +1,5 @@
 <launch>
 
-  <include file="$(find pr2_moveit_config)/launch/demo.launch"/>
-
```

Comment:
> Note the python version of this tutorial now also uses two launch commands: #47

That's cheating, you asked them to change it to fit your request here. ;)

For now, let's keep in mind that this has to be adapted in the other tutorials too.